src/app/src/main/kotlin/org/vaccineimpact/api/app/repositories/jooq/JooqBurdenEstimateRepository.kt

```diff
@@ -479,7 +481,7 @@ class JooqBurdenEstimateRepository(
             this.modelVersion = modelVersionId
             this.responsibility = responsibilityId
             this.uploadedBy = uploader
-            this.uploadedOn = Timestamp.from(timestamp)
```

Comment:
Now that jooq maps sql timestamps to `LocalDateTime`, the conversion to `Instant` is a bit fiddlier. So I added a custom converter to handle the conversion at the code generation level, see comment below.